app/models/pet.rb

```diff
@@ -31,6 +31,7 @@ class Pet < ApplicationRecord
   has_many :adopter_applications, dependent: :destroy
   has_one :match, dependent: :destroy
   has_many_attached :images
+  enum species: ["dog", "cat"]
```

Comment:
Let's add quick test for this using shoulda - https://matchers.shoulda.io/docs/v3.1.1/Shoulda/Matchers/ActiveRecord.html#define_enum_for-instance_method